### DISCUSSION TITLE
Automatized mvn spottless:apply

### DISCUSSION THREAD

**Discussion by enesvarol189** [diffplug/spotless#2295] [Q&A]
Would it be possible to set up a workflow that automatically runs the `mvn spotless:apply` command on the repository whenever a commit is pushed or a pull request is merged into the stage and main branches? Thank you!

**Comment by nedtwigg** [comment]
You could, but it would only help if it

- ran the command
- committed if there was anything to commit
- then pushed

You might want to look at the "CleanThat" GitHub app.
